features/api/vulnerabilities_cve.feature

```diff
@@ -52,6 +52,9 @@ Feature: Client behaviour for CVE vulnerabilities API
               "notes": [],
               "priority": "low",
               "published_at": ".*",
+              "related_packages": [
+                "accountsservice"
+              ],
```

Comment:
This isn't in the spec.

If this data is needed by the CLI, we need to query it specifically for that use case. We shouldn't put it in the API.

Reply:
I don't know if querying that information outside of this endpoint is a good approach, but we can better discuss it our next meeting.